tracing/tracing.hh

```diff
@@ -175,6 +175,7 @@ struct event_record {
     std::string message;
     elapsed_clock::duration elapsed;
     i_tracing_backend_helper::wall_clock::time_point event_time_point;
+    sstring scheduling_group_name = current_scheduling_group().name();
```

Comment:
The commit message says it saves the "service level" but I see here the scheduling group name. I think we should be more consistent in the commit message (these can be different names, right?)

Reply:
This phrasing was used in the original commit that I cherry picked. Should I still change it?

Reply:
I don't know. I'm not even sure my complaint is correct. Do whatever you think is better. If there's an error in the commit message and you can spend 3 minutes to fix it, I think it's worth it.